### ISSUE TITLE
Add cursor metrics to MongoDB plugin

### ISSUE THREAD

**Issue by torrick** [influxdata/telegraf#3480]
## Feature Request

Opening a feature request kicks off a discussion.

### Proposal:
Add Cursor metrics to MongoDb plugin

### Current behavior:
Cursor metrics are not included in the collection process.

### Desired behavior:
Cursor metrics would be included in with the MongoDb collection process.

### Use case: 
Leaking cursors in MongoDB is a fairly common operational problem and it would be nice to collect that metric with telegraf.  We're already collecting many things from `db.serverStatus()` so it would just be including something that's already present in that output.  

```
db.serverStatus().metrics.cursor
{
	"timedOut" : NumberLong(0),
	"open" : {
		"noTimeout" : NumberLong(0),
		"pinned" : NumberLong(0),
		"total" : NumberLong(0)
	}
}
```

**Comment by danielnelson** [comment]
@grubernaut Do you have any thoughts on this?

**Comment by grubernaut** [comment]
@danielnelson sure, I can take a stab at this, depending on $load at work in the next couple days.

**Comment by grubernaut** [comment]
@danielnelson, pushed #4114, sorry for the extra delay in this, $work's been... busy. :smile:

**Comment by danielnelson** [comment]
Thanks @grubernaut, merged for 1.7.0.

@torrick it would be great if you could test it out using the [nightly builds](https://github.com/influxdata/telegraf#nightly-builds) before the release, it should be added to the next build automatically so give it a day.
